shell/platform/darwin/macos/framework/Source/FlutterView.mm

```diff
@@ -98,6 +98,14 @@ - (BOOL)isOpaque {
   return YES;
 }
 
+/**
+ * Declares that the initial mouse-down when the view is not in focus will send an event to the
+ * view.
+ */
+- (BOOL)acceptsFirstMouse {
```

Comment:
The NSView method isn't the one you are overriding here. It accepts and event too: [`- (BOOL)acceptsFirstMouse:(NSEvent *)event;`](https://developer.apple.com/documentation/appkit/nsview/1483410-acceptsfirstmouse?language=objc). Does this do anything?

Reply:
thank you, silly mistake on my part. my unit test passed locally so I assumed it was fine (not sure what I missed there), but that probably explains why the PR checks are failing